test/format/backup.c

```diff
@@ -579,6 +581,8 @@ backup(void *arg)
         /*
          * open_cursor can return EBUSY if concurrent with a metadata operation, retry in that case.
          */
+        trace_msg(session, "Backup #%u start%s%s%s", ++counter, config == NULL ? "" : ": (",
+          config == NULL ? "" : config, config == NULL ? "" : ")");
```

Comment:
Wow - that's hard to read, but I believe it outputs something sensible. Three ternary expressions in a row with the same condition doesn't feel great, but I'm struggling to see a better way.

Reply:
I would change it to something like this, what do you think?
        if (config == NULL)
            trace_msg(session, "Backup #%u start", ++counter);
        else
            trace_msg(session, "Backup #%u start%s%s%s", ++counter, ": (", config, ")");

Reply:
Yeah, it's either an if/else or this.

@AlexeyAnisimovAU, if you go with your change, the constant strings in the `else` shouldn't be arguments, that's confusing.

Reply:
🤷 I'm happy to leave as-is or change out for the if/else. It's a personal preference thing, so I probably shouldn't have mentioned it.

Reply:
Since @agorrod raised this, there's a high chance that somebody else will raise it again later. We probably should change it to if/else to improve readability.

Reply:
I've created a ticket for all the small changes, including this one: [WT-9563](https://jira.mongodb.org/browse/WT-9563)